lib/diagnostic.js

```diff
@@ -47,7 +49,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  alderaan.fire.bind(alderaan);
```

Comment:
`.call` instead here.  One returns a function, one is an execution.  I posted a link in the `links` channel in slack.  Please read it. I think it will clarify.